### ISSUE TITLE
clicking back button on project details page goes to previous page user visited

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/DeVouch-FE#172]
Currently when a user is on the project details page and clicks the back button it always goes to the all projects page. The back button should instead go to the page where the user was last. 

So for example if the user is on My Attestations -> views the details of a project they attested to -> hits the back button it SHOULD go back to the My Attestations page, NOT to the all projects page

![Image](https://github.com/user-attachments/assets/1cbf0f75-2274-41f8-9af8-688e8dcdd5c4)

**Comment by mateodaza** [comment]
yes @divine-comedian not sure why it didn't link the PR properly 
https://github.com/Giveth/DeVouch-FE/pull/182

**Comment by LatifatAbdullahi** [comment]
@divine-comedian @mateodaza 


The back button now works as expected
